### ISSUE TITLE
Ablation experiments learning rate

### ISSUE THREAD

**Issue by fisherman1504** [princeton-vl/WAFT#19]
Thank you for your work.

Currently I have issues reproducing the baseline for the zero-shot ablation experiments (WAFT-DAv2-a1) of Table 5, perhaps you can assist me with this.

If I train for 50k steps on Things, with a batch size of 32 and learning rate 1e−4, the metrics for Sintel are worse than the paper numbers. However, if I change the learning rate to 4e-4 (similar to issue #14 and #18), the numbers get closer to the paper for Sintel.

| Method                       | Sintel train EPE (clean) | Sintel train EPE (final) | Spring sub-val EPE  | Spring sub-val 1px |
| ---------------------------- | ------------------------ | ------------------------ | ------------------- | ------------------ |
| WAFT-DAv2-a1 (paper numbers) | 1.18                     | 2.33                     | 0.27                | 1.43               |
| paper config, lr=1e-4        | 1.4541367368878193       | 2.557698186583581        | 0.2339871915153645  | 1.0128692113633764 |
| paper config, lr=4e-4        | 1.1412928302975427       | 2.384724774382701        | 0.20216778458240603 | 0.8903833587878718 |

Can you clarify what learning rate is intended for the ablation experiments? Thank you.

**Comment by MemorySlices** [comment]
Thank you for pointing this out! In fact, I'm not clear about the reason either. Looks like you can reproduce better results with different learning rate on the same datasets, so it is fine I think : )

**Comment by MemorySlices** [comment]
Hi, I just double checked my code and realized the issue.

The correct learning rate should be 4e-4 instead of 1e-4, with a batch size of 32 (30 is also ok).

Sorry for the typo in the original paper!

**Comment by fealty94** [comment]
Thank you for sharing your reproduction results.
While I was replicating the zero-shot ablation experiments (WAFT-Twins-a2) from Table 4, I noticed a slight discrepancy regarding the hyperparameters.

The paper mentions a step size of 50k, but the provided configuration file (chair-things.json) is set to 100k. Could you please clarify which value was used to achieve the reported results? It seems that 50k might be the intended value based on this issue. Could you confirm if this is correct?

**Comment by MemorySlices** [comment]
> Thank you for sharing your reproduction results. While I was replicating the zero-shot ablation experiments (WAFT-Twins-a2) from Table 4, I noticed a slight discrepancy regarding the hyperparameters.
> 
> The paper mentions a step size of 50k, but the provided configuration file (chair-things.json) is set to 100k. Could you please clarify which value was used to achieve the reported results? It seems that 50k might be the intended value based on this issue. Could you confirm if this is correct?

Yes, you are correct. For "Chairs + Things" zero-shot exps, we train the model for 50k (Chairs) + 50k (Things) steps.
